<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="pic">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>251</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="ppar">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>450</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Описание проекта</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Организация</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Актуальность</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Примеры применения</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Авторы проекта</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Правовая защита</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Экономическая информация</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="pname">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Название проекта</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchbar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asearch">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Расширенный поиск</string>
    </property>
   </widget>
   <widget class="QComboBox" name="language">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Русский</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
   </widget>
   <widget class="QScrollArea" name="plist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>491</width>
      <height>221</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>489</width>
       <height>219</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="searchfield" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>481</width>
      <height>165</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="orgpr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="orp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Название организации</string>
     </property>
    </widget>
    <widget class="QLabel" name="authp">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Авторы проекта</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="autpr">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="invest">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Сумма инвестиций</string>
     </property>
    </widget>
    <widget class="QLabel" name="finv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>от</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inv1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="tinv">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>до</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inv2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="cat">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Категория</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="categor">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="datedev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата завершения проекта</string>
     </property>
    </widget>
    <widget class="QLabel" name="fddev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>от</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="datedf">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="tddev">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>до</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="datedt">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dateinv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>122</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата оккупаемости проекта</string>
     </property>
    </widget>
    <widget class="QLabel" name="fdinv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>от</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateif">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="tdinv">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>до</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="stage">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Стадия проекта</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stag">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="categ">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Категория</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отменить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
